glib/example/vala/sqlite.vala

```diff
@@ -0,0 +1,33 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+
+int main (string[] args) {
+    try {
+    	var database = new GADBC.Database ();
+    	database.set_option ("driver", "adbc_driver_sqlite");
+  	database.set_option ("uri", "test.db");
+    	database.init ();
+    	var conn = new GADBC.Connection ();
+    	conn.init (database);
+    }
+    catch (GLib.Error e) {
+    	GLib.message ("Error: %s", e.message);
```

Comment:
```suggestion
    	GLib.message("Error: %s", e.message);
```

Reply:
In Vala code is more common to use spaces before the parenthesis, so do you want this examples follow your conventions?

Reply:
Oh, really!? Could you tell us a URL of a Vala's common coding style guideline?

Reply:
You can go to: https://vala.dev/

Reply:
Do you know a coding style guideline page? Or should I find it from the site by myself?

Reply:
> Do you know a coding style guideline page? Or should I find it from the site by myself?

There aren't a Vala specific code style, but as described at [Vala Conventions](https://www.vala-project.org/doc/vala/Overview.html#Vala_conventions), it follows [GLib ones](https://developer.gnome.org/documentation/guidelines/programming/coding-style.html). A Vala specific code style, can be found at [elementaryOS Code Style](https://docs.elementary.io/develop/writing-apps/code-style).

In all cases, the use of whitespace after the function name is an accepted convention.

So, even the GLib/C example, could use [GLib ones](https://developer.gnome.org/documentation/guidelines/programming/coding-style.html) so the users can copy and paste the code in their projects easy.

Reply:
Thanks. It's what I want to know.
I found that there is a linter based on the elementary's coding style: https://github.com/vala-lang/vala-lint

We're using `clang-format` for C/C++ files:
https://github.com/apache/arrow-adbc/blob/c78398bb7b76edf67ac456fca7aafcaebb514ce8/.pre-commit-config.yaml#L42-L47

We can use vala-lint for Vala files to use the elementary's coding style.

Reply:
I've opened a pull request to Vala-Lint for pre-commit support: https://github.com/vala-lang/vala-lint/pull/179